server/resources/resource_management_test.go

```diff
@@ -1292,6 +1305,27 @@ var _ = Describe("Resource manager", func() {
 					Expect(ok).To(BeTrue())
 					Expect(theResource).To(HaveKeyWithValue("test_string", "Ia, ia, HJPEV fhtang!"))
 				})
+
+				It("Should only modify updated fields", func() {
+					networkSchema, _ := manager.Schema("network")
+
+					err := resources.CreateResource(context, testDB, fakeIdentity, networkSchema, adminNetworkData)
+					Expect(err).ToNot(HaveOccurred())
+					result := context["response"].(map[string]interface{})
```

Comment:
result is unused - and i think it could be good idea to check its contents (i.e. `Expect(result).To(Equal))`) as it'll be easier to spot which part of network has been updated

Reply:
good point, will fix